### DISCUSSION TITLE
How to Share the T-Pot Dashboard with Others Without Giving away your Honeypot

### DISCUSSION THREAD

**Discussion by jcilley-mhvfcu** [telekom-security/tpotce#1386] [Q&A]
Quick question for the community - Has anyone come up with a way to share the Tpot Kibana dashboard or some other dashboard publicly without giving away the identity of your honeypot or potentially inviting attacks against the administrative interfaces within the platform?

I run T-pot honeypot in a standalone configuration and would like to share the data with my peers. However, I don't want to potentially expose my install to others in terms of giving them access directly into the web interface.

Thank you!

**Comment by t3chn0m4g3** [comment]
ELK in T-Pot does not offer user management.

**Reply by jcilley-mhvfcu** [reply]
That's kind of what I figured, but was curious to know if anyone had been able to come up with some kind of alternative access solution.

I could probably set something up to scrape the T-Pot Kibana dashboard periodically and post it to a slack channel internally which would do the trick for now.
